sources/packages/backend/libs/integrations/src/services/sfas/sims-to-sfas.service.ts

```diff
@@ -119,6 +127,128 @@ export class SIMSToSFASService {
     return modifiedStudentIds;
   }
 
+  /**
+   * Get all student ids and application data of students who have one or more updates in application related data.
+   * @param modifiedSince the date after which the application data was updated.
+   * @param modifiedUntil the date until which the application data was updated.
+   */
+  async getAllApplicationsWithUpdates(
+    modifiedSince: Date,
+    modifiedUntil: Date,
+  ): Promise<ApplicationRecord[]> {
+    return (
+      this.applicationRepo
+        .createQueryBuilder("application")
+        .select("application.id", "applicationId")
+        .addSelect("student.id", "studentId")
+        .addSelect("programYear.programYear", "programYear")
```

Comment:
Please request business to update the type of program year field in documentation. 

![image](https://github.com/user-attachments/assets/ca48c8e0-32e6-442c-8ab3-8ead099f8e21)